launcher/Application.cpp

```diff
@@ -502,7 +503,7 @@ Application::Application(int &argc, char **argv) : QApplication(argc, argv)
 
         // Theming
         m_settings->registerSetting("IconTheme", QString("pe_colored"));
-        m_settings->registerSetting("ApplicationTheme", QString("system"));
+        m_settings->registerSetting("ApplicationTheme");
```

Comment:
nit: to keep it consistent with other settings

```suggestion
        m_settings->registerSetting("ApplicationTheme", QString());
```